airflow/dag/serialization/json_schema.py

```diff
@@ -40,6 +51,15 @@ def iter_errors(self, instance) -> Collection[jsonschema.exceptions.ValidationEr
 
 
 def load_dag_schema() -> Validator:
-    schema = json.loads(pkgutil.get_data(__name__, 'schema.json'))
+    """
+    Load Json Schema for DAG
+    """
+    schema_file_name = 'schema.json'
+    schema_file = pkgutil.get_data(__name__, schema_file_name)
+
+    if schema_file is None:
+        raise AirflowException("Schema file {} does not exists".format(schema_file_name))
```

Comment:
We don't need this - it will already throw an exception if it's not found. And they only way this can happen is if someone _really_ messes with their python install

Reply:
`pkgutil.get_data` returns `None` if file is not found. And it also gives mypy error

Reply:
It does? I changed it to `schema.jsoan` and Got: `FileNotFoundError: [Errno 2] No such file or directory: '/Users/ash/code/python/incubator-airflow/airflow/dag/serialization/schema.jsoan`

Still, this doesn't hurt.